proposals/compatibility-mode.md

```diff
@@ -197,7 +197,19 @@ generate a validation error.
 other APIs only support the first vertex so only `@interpolation(flat, either)` is supported in
 compatibility mode.
 
-## 18. Introduce new `maxStorageBuffersInVertexStage` and `maxStorageTexturesInVertexStage` limits.
+## 18. Introduce new `float16-renderable` and `float32-renderable` features.
+
+When supported, `float16-renderable` allows the `RENDER_ATTACHMENT` usage on textures with format `"r16float"`, `"rg16float"`, and `"rgba16float"`.
+
+When supported, `float32-renderable` allows the `RENDER_ATTACHMENT` usage on textures with format `"r32float"`, `"rg32float"`, and `"rgba32float"`.
+
+Without support, an error will occur at texture creation time as described in section 6.1.3.
+
+Support for both features is mandatory in core WebGPU.
```

Comment:
@SenorBlanco @kainino0x I have a question regarding how these optional features in compat / required in core will work.

Does that mean a core implementation must return these features as part of [GPUAdapter.features](https://www.w3.org/TR/webgpu/#gpuadapter)?

Or should instead if [requestAdapter](https://www.w3.org/TR/webgpu/#dom-gpu-requestadapter) returns a core adapter, the above features are implied and can not be requested?

I think we should prefer the latter as otherwise we will cause breaking changes to core when a website adopts compat mode, which does not seem to be the intent of compat.